Common/src/main/java/com/tabnineCommon/chat/commandHandlers/ChatMessageHandler.kt

```diff
@@ -18,7 +18,7 @@ abstract class ChatMessageHandler<RequestPayload, ResponsePayload>(protected val
     }
 
     protected fun getEditorFromProject(project: Project): Editor? {
-        return AsyncAction(ActionPermissions.WRITE).execute {
+        return AsyncAction(ActionPermissions.READ).execute {
```

Comment:
@yairco1990 why not reuse the `getActiveEditor` method that is used in the completions? 
https://github.com/codota/tabnine-intellij/blob/ee546ce2ea61b06e3f0662b965896ea3ae73ba38/Common/src/main/java/com/tabnineCommon/inline/TabnineDocumentListener.java#L95

Reply:
the completions approach also uses EDT which can lead to the same problem